### PR TITLE
Js green integration

### PR DESCRIPTION
**Pull request by jsotobroad**

### CODE REVIEW

File: src/main/resources/integrationTestCases/green/singlesample/singlesample.wdl [hunk 1]
```diff
@@ -0,0 +1,1661 @@
+## Copyright Broad Institute, 2016
```
Comment (mcovarr) [inline src/main/resources/integrationTestCases/green/singlesample/singlesample.wdl:1]: 2017?

**Comment by mcovarr** [comment]
:+1: wow real workflows are a lot more intense than toy workflows

[Approved with PullApprove](https://pullapprove.com/broadinstitute/centaur/pull-request/213/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=broadinstitute/centaur)

**Review by mcovarr — APPROVED** [review]

**Comment by geoffjentry** [comment]
@jsotobroad Just double checking that everything here is OK to live in a public repo

File: src/main/resources/integrationTestCases/green/singlesample/singlesample.wdl (hunk 1 above)
Comment (jsotobroad) [reply src/main/resources/integrationTestCases/green/singlesample/singlesample.wdl:1]: Yea I'm not sure, this is just the same wdl GOTC uses.  I don't think it matters to us but will add this to the "can this be public" question

**Comment by jsotobroad** [comment]
I *thought* we made our full wdl public including metrics and whatnot but will double check with @ktibbett just in case

**Comment by Horneth** [comment]
👍 

[Approved with PullApprove](https://pullapprove.com/broadinstitute/centaur/pull-request/213/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=broadinstitute/centaur)

**Comment by jsotobroad** [comment]
@geoffjentry - @ktibbett gave me a verbal thumbs for these version of the wdls.

**Pull request merged**